### PR TITLE
add chart & update gcloud project

### PR DESCRIPTION
**Pull request by roberthuskins**
Screen Shot 2020-03-07 at 9 31 22 PM

SPS Week 3 (Libraries)
Website now displays a chart of the most watched youtube videos (Wikipedia)

### CODE REVIEW

File: portfolio/src/main/webapp/charts.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  const options = {
+    'title': 'Most viewed YouTube videos',
+    'width':600,
+    'height':400
+  };
+
+  const chart = new google.visualization.BarChart(
+      document.getElementById('chart-container'));
+  chart.draw(data, options);
+}
```
Comment (IngridFeng) [inline portfolio/src/main/webapp/charts.js]: nit: Add newline at end of file

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -29,6 +31,8 @@
           
       </div>
       <br>
+      <div id="chart-container"></div>
+      <br>
```
Comment (IngridFeng) [inline portfolio/src/main/webapp/index.html]: I might be wrong - my understanding is that the `<br>` here is just to add white space in pages.
Using `<br>` to add space is usually bad practice. It's better to use margin/padding i.e. you needs to change css.
Please look into how to add css for an element of a specific id.

Comment (roberthuskins) [reply portfolio/src/main/webapp/index.html]: Thank you. I changed the CSS a bit, could you take a look?

Comment (IngridFeng) [reply portfolio/src/main/webapp/index.html]: Great job!!

**Pull request merged**